tests/entrypoints/test_openai_server.py

```diff
@@ -676,6 +676,22 @@ async def test_response_format_json_object(server, client: openai.AsyncOpenAI):
     assert loaded == {"result": 2}, loaded
 
 
+async def test_complex_message_content(server, client: openai.AsyncOpenAI):
+    resp = await client.chat.completions.create(model=MODEL_NAME,
+                                                messages=[{
+                                                    "role":
+                                                    "user",
+                                                    "content": [{
+                                                        "type":
+                                                        "text",
+                                                        "text":
+                                                        "what is 1+1?"
+                                                    }]
+                                                }])
+    content = resp.choices[0].message.content
+    assert "2" in content
```

Comment:
`ground_truth = ....`
Can we directly `content == ground_truth`?

Reply:
Not sure, the model will probably give slightly different answers. It's also not really important, we mostly want to ensure that the request is accepted.